profiling/simple-kernel-timer/kp_kernel_info.h

```diff
@@ -218,13 +214,13 @@ class KernelPerformanceInfo {
 
   char* kernelName;
   // const char* regionName;
-  uint64_t callCount;
-  double time;
-  double timeSq;
-  double startTime;
+  uint64_t callCount = 0;
+  double time        = 0;
+  double timeSq      = 0;
+  double startTime   = 0;
```

Comment:
How are these changes related to sharing the implementation of `find_index`?

Reply:
Indeed, reverted :+1: